docs_src/microservices/V2TopLevelMigration.md

```diff
@@ -0,0 +1,97 @@
+
+!!! note
+    Since the V1 data and V2 data are stored separately, one could create a migration tool and upstream it to the EdgeX community.
+
+!!! warning
+    If the database is not cleared before starting the V2 services, the old V1 data will still reside in the database taking up useful memory. It is recommended that you first wipe the database clean before starting EdgeX 2.0 Services. That is unless you create a DB migration tool.
+
+The following sections describe what you need to be aware for the different services that use the database.
+
+### Core Data
+
+The Event/Reading data stored by Core Data is considered transient and of little value once it has become old. The V2 versions of these data collections will be empty until new Events/Readings are received from V2 Device Services. 
+
+### Core Metadata
+
+The data stored by Core Metadata will be recreated when the V2 versions of the Device Services start-up. The statically declared devices will automatically be created and device discovery will find and add existing devices. Any devices created manually via the V1 REST API will have to be recreated using the V2 REST API.
+
+### Core Command
+
+Core Command in V2 no longer uses the database. It now retrieves the needed information from Core Metadata REST APIs.
+
+### Support Notifications
+
+Any `Subscriptions` created via the V1 REST API will have to be recreated using the V2 REST API. The `Notification` and `Transmission`collections will be empty until new notifications are sent using EdgeX 2.0 
+
+### Support Scheduler
+
+The statically declared `Interval` and `IntervalAction` will be created automatically. Any `Interval` and/or `IntervalAction` created via the V1 REST API will have to be recreated using the V2 REST API. 
+
+### Application Services
+
+Application services use the database only when the [Store and Forward](../application/AdvancedTopics/#store-and-forward) capability is enabled. If you do not use this capability you can skip this section. This data collection only has data when that data could not be exported. It is recommended not to upgrade to V2 while the Store and Forward data collection is not empty or you are certain the data is no longer needed. You can determine if the Store and Forward data collection is empty by setting the Application Service's log level to `DEBUG`  and look for the following message which is logged every `RetryInterval`:
+
+```tex
+msg=" 0 stored data items found for retrying"
+```
+
+## Custom Configuration
+
+### Consul
+
+If you have customized any EdgeX service's configuration  (core, support, device, etc.)  via Consul, those customization will need to be re-applied to those services' configuration in Consul once the V2 versions have started and pushed their configuration into Consul. The V2 services now use `2.0` in the Consul path rather than `1.0` .
```

Comment:
Reference the V2 Migration of common configuration here??

Reply:
Sure I can add that reference and to the TOML File section for the service specific changes